.github/CODEOWNERS

```diff
@@ -4,5 +4,4 @@
 # most precedence.
 
 # Primary repo maintainers
-*       @aaronc @alexanderbez
-
+*       @cosmos/sdk-core-dev
```

Comment:
GitHub is complaining about this. I believe this is because the upstream team is not public.
We likely should not mention the upstream team anyway.

> Unknown owner on line 7: make sure the team `@cosmos/sdk-core-dev` exists, is publicly visible, and has write access to the repository
> *       `@cosmos/sdk-core-dev`

Reply:
Yeah, we might want to fix that later. I'll confer with @michaelfig